src/lib/y2network/interface_config_builder.rb

```diff
@@ -528,5 +538,13 @@ def ip_config_default
       end
       default
     end
+
+    # method that allows easy change of backend for providing data
+    # it also logs error if result differs
+    def select_backend(old, new)
```

Comment:
do we need such "simple" backend access? I mean when we switch backends we have to reload whole config anyway and definitely cannot be done during life of the builder (adding, editing iface) in a simple way.

Reply:
this method is intended mainly for testing. You can easily change what it returns and it also logs if values in backend differs. As you can you in my previous PR, currently I write values to both backends. So it hope it should be possible to read it from both backends and compare result and also test it in UI by simply changing this method.
It is not something for production.

Reply:
then it deserve a "todo" or "fixme"

Reply:
We might even use an (undocumented) environment variable, so you do not need to modify this method when testing.

Reply:
We might even use an (undocumented) environment variable, so you do not need to modify this method when testing.

Reply:
yes, it is also option